### PR TITLE
add chords to planning dummy client

### PR DESCRIPTION
**Pull request by Juphex**
Add regex for splitting the notes including only the notes from the scale (C, D, E, F, G, A, B)

 e.g. 'C, <C#, D>, C#, A' => ['C', '<C#, D>', 'C#', 'A']

Updating the dummy client from @Flova

### CODE REVIEW

File: marimbabot_planning/scripts/dummy_client.py [hunk 1]
```diff
@@ -21,19 +26,43 @@ def main_loop(self):
...
+            input_str = input("Enter a sequence of hits (e.g. 'C, <C#, D>, C#, A'): ")
             goal = HitSequenceGoal()
             start_time = rospy.Time(0)
             duration = rospy.Duration(0.5)
-            for note in input_str.split(','):
+
+            # create regex for splitting the notes including only the notes from the scale (C, D, E, F, G, A, B)
+            # e.g. 'C, <C#, D>, C#, A' => ['C', '<C#, D>', 'C#', 'A']
+            # LIMITED TO A CHORD OF 2 NOTES !!!
+            regex_pattern = r'([CDEFGAB][#b]?|[<].*?[>])'
```
Comment (ateRstones) [inline marimbabot_planning/scripts/dummy_client.py:38]: I think you should make this regex more restricting.
Currently the inside part of the brackets is not checked and you can put in however many and any you symbols you want. I think this should be checked
```
>>> re.findall(regex_pattern, "C, <C#, D>, C#, A")
['C', '<C#, D>', 'C#', 'A']
>>> re.findall(regex_pattern, "C#, <C#, H, D>, C#, A")
['C#', '<C#, H, D>', 'C#', 'A']
>>> re.findall(regex_pattern, "C#, <C#, Hb, D>, C#, A")
['C#', '<C#, Hb, D>', 'C#', 'A']
>>> re.findall(regex_pattern, "C#, <C#, Hello good sir nice to see you, D>, C#, A")
['C#', '<C#, Hello good sir nice to see you, D>', 'C#', 'A']
```

Comment (ateRstones) [inline marimbabot_planning/scripts/dummy_client.py:38]: The parser also doesn't check the ,
```
>>> re.findall(regex_pattern, "C#AbC")
['C#', 'Ab', 'C']
```

is this desired?

Comment (Juphex) [reply marimbabot_planning/scripts/dummy_client.py:38]: Sure, I mean the desired input is prompted and it the code is just for sending sequences without starting the behavior node, but I can do that.

Comment (Juphex) [reply marimbabot_planning/scripts/dummy_client.py]: No, I don't think that harms a lot anyways

**Review by Flova — APPROVED** [review]

**Pull request merged**